chrome/extension/devtools.js

```diff
@@ -197,7 +202,32 @@ chrome.devtools.inspectedWindow.onResourceAdded.addListener(
   sendStyleDiffChangedEvent
 );
 
-// chrome.devtools.inspectedWindow.onResourceAdded.addListener(sendContentStyles);
+chrome.devtools.inspectedWindow.onResourceContentCommitted.addListener(
+  resource => {
+    if (resource.url.startsWith("debugger://")) {
```

Comment:
Shouldn't we also filter to only stylesheets?

Reply:
No - inline <style> tags are type document instead of type stylesheet. Filtering to stylesheet would break that. I guess we could do !== script, what do you think?

Reply:
you're sure that this won't trigger whenever HTML changes due to like, React re-rendering?

Reply:
Yes because it's `onResourceContentCommitted`